cadence/contracts/bridge/CrossVMNFT.cdc

```diff
@@ -28,17 +30,24 @@ access(all) contract CrossVMNFT {
 
     /// Proof of concept metadata to represent the ERC721 values of the NFT
     ///
-    access(all) struct BridgedMetadata {
-        access(all) let name: String
-        access(all) let symbol: String
-        access(all) let uri: URI
-        access(all) let evmContractAddress: EVM.EVMAddress
+    access(all)
+    struct EVMBridgedMetadata {
```

Comment:
Note that this and `URI` have been raised for feedback and as potential additions to `MetadataViews`